examples/animation/strip_chart.py

```diff
@@ -49,14 +49,14 @@ def emitter(p=0.03):
             yield np.random.rand(1)
 
 # Fixing random state for reproducibility
-np.random.seed(19680801)
+np.random.seed(1968080)
```

Comment:
```suggestion
np.random.seed(19680801)
```

One of the sentimental things we do is seed the random numbers to John Hunter's birthday.  Would rather not change this unless there is a compelling reason.

Reply:
This was intentional -- the original seed did not yield many spikes for the given probability (and increasing the probability further made too many spikes) over the first few seconds of the animation

Reply:
How about I write it as `19680801 // 10`? Same effect but preserves the birthday in there?

Reply:
(Or I could test if 0 or 41 or 1337 or some other "principled" choice of seed gives something that plays well in the animation)

Reply:
`19680801 // 10` <- if that looks good :+1:

Could also "prime" the stream a bit by generating and discarding a bunch of numbers?

Reply:
Pushed the ` // 10` let's see if CircleCI produces something reasonable -- it should!